specification/resources/resource-manager/Microsoft.Authorization/preview/2023-03-01-preview/privateLinks.json

```diff
@@ -0,0 +1,834 @@
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete a Resource Management Private Link.": {
+            "$ref": "./examples/ListSubscriptionResourceManagementPrivateLink.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Authorization/resourceManagementPrivateLinks": {
+      "get": {
+        "tags": [
+          "ResourceManagementPrivateLink"
+        ],
+        "operationId": "ResourceManagementPrivateLink_ListByResourceGroup",
+        "description": "Get all the resource management private links in a resource group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group. The name is case insensitive.",
+            "minLength": 1,
+            "maxLength": 90
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok - Returns an array of private links.",
+            "schema": {
+              "$ref": "#/definitions/ResourceManagementPrivateLinkListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete a Resource Management Private Link.": {
+            "$ref": "./examples/ListResourceGroupResourceManagementPrivateLink.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Authorization/enablePrivateLinkNetworkAccess": {
+      "post": {
+        "tags": [
+          "EnablePrivateLinkNetworkAccess"
+        ],
+        "operationId": "EnablePrivateLinkNetworkAccess_Post",
+        "description": "Enable private link network access for global administrator",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/EnablePrivateLinkNetworkAccessObject"
+            },
+            "description": "Parameters supplied to enable private link network access."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Post Succeed."
+          },
```

Comment:
Does it suppose to return empty response body? If yes, use 204 response instead of 200 response, as the 200 response can't have empty body.